parser/scanner.go

```diff
@@ -107,7 +134,7 @@ func (s *Scanner) EatString(str string, eaten *Scanner) bool {
 
 // EatRegexp eats the text matching a regexp, populating match (if != nil) with
 // the whole match and captures (if != nil) with any captured groups. Returns
-// n as the number of captures set and ok iff a match was found.
+// n as the number of captures set and ok if a match was found.
```

Comment:
`iff` is shorthand for `if and only if`.

Reply:
😯